test/extensions/tracers/skywalking/skywalking_client_config_test.cc

```diff
@@ -0,0 +1,97 @@
+#include "extensions/tracers/skywalking/skywalking_client_config.h"
+
+#include "test/mocks/common.h"
+#include "test/mocks/server/tracer_factory_context.h"
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using namespace testing;
```

Comment:
style: `using namespace` is prohibited per style guide.

Reply:
Ah, it's my fault.